grails-app/controllers/org/pih/warehouse/order/PurchaseOrderController.groovy

```diff
@@ -86,7 +86,7 @@ class PurchaseOrderController {
             return
         }
 
-        redirect(action: addItems, id: order.id)
+        redirect(action: "addItems", id: order.id)
```

Comment:
Wow I had to look this up. I know in Grails 1.3.9 you can pass a closure to "action" but I had never seen it done and wasn't sure of the 

> action - The action to redirect to, either a string name or a reference to an action within the current controller

https://grails.github.io/grails2-doc/1.3.9/ref/Controllers/redirect.html

So in grails 3.3.10, where actions are no longer controllers (but methods) ... I suspect this isn't ok any longer. The docs for the redirect's action parameter suggest it is no longer ok.

> action (optional) - the name of the action to use in the link, if not specified the default action will be linked

But the full documentation implies that it is still ok. 
https://docs.grails.org/3.3.10/guide/single.html#_chaining

Reply:
So @kchelstowski did you change this because it caused an issue or because it didn't look right.

If we change it here we should probably consider changing it in develop as well. 

@awalkowiak Thoughts?

Reply:
@kchelstowski Ignore, just reread the summary and saw you addressed this.

Reply:
@jmiranda wanted to make screenshots for you from the IntelliJ, but it doesn't want to capture this, so:
I changed it, because as I wrote - in Grails 3 it is supposed to be as string. If it is not as string, even IDE has it in gray and says: `No candidates found for method call addItems.` - which doesn't happen in Grails 1.3.9 - it recognizes the method  both when being in string and not.